src/timers.h

```diff
@@ -10,6 +10,12 @@
 #ifndef TIMERS_H
 #define TIMERS_H
 
+// Timer enumeration
+enum { DATABASE_WRITE_TIMER, EXIT_TIMER, GC_TIMER, LISTS_TIMER, REGEX_TIMER, ARP_TIMER, LAST_TIMER };
+
+// Prepare timers, used mainly for debugging purposes
+#define NUMTIMERS LAST_TIMER
```

Comment:
Not sure that this comment is relevant for this line. Is this a copy-paste error?